### ISSUE TITLE
how to deal with 9 tuplets

### ISSUE THREAD

**Issue by Aphcity** [Audiveris/audiveris#517]
just the kind of tuplet in this score
![Uploading Aimer - Fatestay night Heaven's Feel Medley_页面_01.png…]()

**Comment by Bacchushlg** [comment]
please also have a look at #74

**Comment by hbitteur** [comment]
Yes, we really need some help from the user, the engine will not be able to handle all these complex cases on its own.
 I think Maxim's proposal on #74 looks like the way to go.
